src/workloads/issues/ConnectionsBuildup.yml

```diff
@@ -21,10 +23,13 @@ phasePath: &phasePath ../../phases/issues/ConnectionsBuildup.yml
 # Operations reused in multiple Phases.
 Nop: &Nop {Nop: true}
 EnableSharding: &EnableSharding
+  # Until EVG-21054 is resolved, using OnlyRunInInstance requires excluding the workload from dry-runs.
```

Comment:
It makes more sense to have this comment in dry_run.py

Reply:
Done